processor/transaction/payload.go

```diff
@@ -65,6 +67,10 @@ func (pa *payload) transform(config config.Config) []beat.Event {
 
 	var events []beat.Event
 	for _, event := range pa.Events {
+		var zeroTime = time.Time{}
+		if event.Timestamp == zeroTime {
```

Comment:
Looks like this `IsZero` method could be useful here: https://golang.org/pkg/time/#Time.IsZero